### ISSUE TITLE
Some transitions have several inputs

### ISSUE THREAD

**Issue by mattelothere** [hickatheworld/SM402I-Project#16]
Choose to work with automaton 41, and open the .txt file that is created.
Look at line 49 and you will see : 
```
...
        },
        {
            "from":"3",
            "input":[
                "a",
                "b"
            ],
            "to":"5"
        },
        {
...
```

![image](https://github.com/hickatheworld/SM402I-Project/assets/90071735/2c848ee4-92a3-4220-ba54-0dcbf2db8146)

**Comment by mattelothere** [comment]
i fixed it by modifying the json by hand but idk in which commit lol

**Comment by mattelothere** [comment]
Fixed by comit bda21ea
